src/librustc_const_math/lib.rs

```diff
@@ -29,13 +29,7 @@ extern crate syntax;
 extern crate serialize as rustc_serialize; // used by deriving
 
 mod float;
-mod int;
-mod usize;
-mod isize;
 mod err;
 
 pub use float::*;
```

Comment:
Do we need the `ConstFloat` wrapper anymore? I'd like miri to call into `rustc_apfloat` directly.

Reply:
That would require us to replicate everything happening here in miri. I just checked all occurrences of `ConstFloat`, and they always pass through a FloatTy + bits.

Reply:
I think it's pretty straight-forward to adapt the macros to miri, so you wouldn't have really verbose code or anything like that.